etc/picongpu/wexac-weizmann/gpu_picongpu.profile.example

```diff
@@ -0,0 +1,73 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send mails on job (-B)egin, Fi(-N)ish
+export MY_MAILNOTIFY=""
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module load gcc/6.3.0
+module load cmake/3.18.4
+module load openmpi/2.0.1
+module load cuda/9.2
+module load boost/1.69.0
+
+export CXX=$(which g++)
+export CC=$(which gcc)
+
+# Other Software ##############################################################
+#
+# not yet as module avaialable
+
+# Environment #################################################################
+#
+export PICSRC=$HOME/src/picongpu
+export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PIC_BACKEND="cuda:70"
+
+export PATH=$PATH:$PICSRC
+export PATH=$PATH:$PICSRC/bin
+export PATH=$PATH:$PICSRC/src/tools/bin
+
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+# "tbg" default options #######################################################
+# currently the submit script, generated by tbg, needs to be streamed to bsub
+export TBG_SUBMIT="echo 'manually execute: bsub < '"
```

Comment:
Does having `export TBG_SUBMIT="bsub <"` not work?

Reply:
I am not well familiar with internals of tbg itself, so do not know it.

Reply:
Good point :+1: - I have no access to test this. @danlevy100 could you test this?

Reply:
IMO this line should be 

```
export TBG_SUBMIT="bsub"
```

Maybe the line above is generating a valid example but I do not understand why you would do it instead of executing bsub directly.

Reply:
@psychocoderHPC We have to do that, because the admins of the wexac cluster, as far as we understood it, prevent using the input file option and only allow to "stream" to `bsub`.
@danlevy100 Did that configuration change in the mean time?

Reply:
@PrometheusPi @psychocoderHPC  No, the configuration did not change.
The only way I could get tbg to submit is by "bsub < submit.start".
There is some information here: https://www.ibm.com/docs/en/spectrum-lsf/10.1.0?topic=bsub-write-job-scripts

I had to change `~/src/picongpu/tbg` in order to be able to submit with tbg, as mentioned in one of my comments in the #3496 thread.

Reply:
@sbastrakov According to @danlevy100 post [here](https://github.com/ComputationalRadiationPhysics/picongpu/issues/3496#issuecomment-780655127), `"bsub <"` should not work. @danlevy100 How did you change `tbg`? (Could you provide a `diff`?